std/concurrency.d

```diff
@@ -498,6 +501,12 @@ if (isSpawnable!(F, T))
     assert(receiveOnly!string == "This is so great!");
 }
 
+void spawn(F, T...)(F fn, T args)
```

Comment:
Could you explain why use `void` instead of `Tid` ?

Reply:
Because it's going to fail anyway, I guess. I didn't really think about it. Mayhap `Tid` would be better.